package.json

```diff
@@ -35,7 +37,7 @@
     "seed:run:test": "NODE_ENV=testing knex seed:run",
     "db:reset": "npm run migrate:teardown && npm run migrate:latest && npm run seed:run",
     "db:reset:test": "npm run migrate:teardown:test && npm run migrate:latest:test && npm run seed:run:test",
-    "cover": "jest --coverage",
+    "cover": "jest --runInBand --coverage",
```

Comment:
I don't think we want to run everything in series. Can we separate out commands for those tests that run in series (basically anything touching the DB) and those in parallel (unit tests and others with limited dependencies)

Reply:
We need to run all of our tests together to get an accurate coverage report. As the database tests need to be run in series, this is needed here.

Please add an issue to JEST to ask them why there is no configurable option for specific tests to be run in series.